prReview.js

```diff
@@ -0,0 +1,7 @@
+function findFolderByName(n) {
```

Comment:
**suggestion _(non-blocking)_:** "getFolder" could be a better function name. Besides, since parameter "n" means "name", let's write it in full.